contrib/tools/config-docs-generator/templates/reference_template.md

```diff
@@ -0,0 +1,11 @@
+# Stacks Node Configuration Reference
```

Comment:
I like this! it would be trivial to pull in other templates (even not in this repo) if we needed a specifically formatted markdown (i.e. gitbook).